### PR TITLE
feat(suite-native): Asset detail refactoring

### LINKED ISSUE #6704: Polishing My asset detail UI
Please adjust the My asset detail page according to design in Figma

### LINKED ISSUE #6836: Show message when there is no transaction on asset detail page
As a user I would like to see message when there is no transactions yet on my asset in order not to confused that the page is blank. 

Acceptance criteria:

- If there are no transactions on the asset, display the message
- There is button with Receive which leads to Receive flow

Image

### PR DESCRIPTION
**Pull request by juriczech**
## Related Issue
Resolve https://github.com/trezor/trezor-suite/issues/6836
Resolve https://github.com/trezor/trezor-suite/issues/6704

## Screenshots (if appropriate):
https://tppr.me/G7wzx

https://user-images.githubusercontent.com/36101761/200544309-95b35c97-2f1e-48b6-b987-be7cb90bdb18.mov

### CODE REVIEW

File: suite-native/atoms/src/Card.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ type CardProps = {
 
 const cardStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.gray0,
-    borderRadius: 12,
+    borderRadius: utils.borders.radii.large / 2,
```
Comment (Nodonisko) [inline suite-native/atoms/src/Card.tsx:13]: maybe we should consider redefining border radiuses in theme since we are constantly bending them like this... definitely not part of this PR but something that could be discussed with UI team because they have some consistent rule for it

Comment (juriczech) [reply suite-native/atoms/src/Card.tsx:13]: Good point! I think also adding some smaller spacings (extraSmall = 4) would be additional thing to discuss as well. I will just write a TODO for myself to discuss and possibly update theme.

File: suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                iconName="chevronLeft"
+                onPress={() => navigation.goBack()}
+            />
+            <Text>{accountName}</Text>
+            <IconButton
+                colorScheme="gray"
+                isRounded
+                size="large"
+                iconName="settings"
+                onPress={() => {}}
```
Comment (Nodonisko) [inline suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx]: should we have this here if it is not working? I would vote for removing it until it will be used, otherwise it could be quite confusing for users. Maybe @mnuky should decide?

File: suite-native/module-accounts/src/components/TransactionListHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    accountKey: AccountKey;
+};
+
+export const TransactionListHeader = memo(({ accountKey }: AccountDetailHeaderProps) => {
+    const [_, setIsSendReceiveActionsVisible] = useAtom(isSendReceiveActionsVisibleAtom);
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+
+    const accountHasTransactions = !!account?.history.total ?? false;
```
Comment (Nodonisko) [inline suite-native/module-accounts/src/components/TransactionListHeader.tsx]: this should be moved to selector because it will be useful in more places than just here for sure

File: suite-native/navigation/src/components/TabBar.tsx [hunk 4]
```diff
@@ -36,18 +39,20 @@ export const TabBar = ({
     tabItemOptions,
     SendReceiveComponent,
 }: TabBarProps) => {
-    const [sendReceiveActionsVisible, setSendReceiveActionsVisible] = useState(false);
+    const [isSendReceiveActionsVisible, setIsSendReceiveActionsVisible] = useAtom(
```
Comment (Nodonisko) [inline suite-native/navigation/src/components/TabBar.tsx:42]: I really love how simple and clean this is compared to redux solution, definitely good way to use jotai instead.

**Review by Nodonisko — APPROVED** [review]

File: suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx (hunk 2 above)
Comment (mnuky) [reply suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx]: is it the settings button on top right? If yes, remove it, until we do the Asset settings feature.

Comment (juriczech) [reply suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx]: Daaamn. I think it will be easier to just create the screen with some super simple option maybe at least to delete the xpub. Because if I remove it, I will have to restyle the header and then restyle it again when we create the screen 😅😭

Comment (Nodonisko) [reply suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx]: We should have some kind of header component that would not break always when we add or remove icon in first place, because we have this problem nearly everywhere.

Anyway for now, I will be okay with something like `opacity: 0` with explanantion comment why it is here.

File: suite-native/transactions/src/components/TransactionsEmptyState.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 1,
+    width: '100%',
+    paddingHorizontal: utils.spacings.large,
+    borderRadius: utils.borders.radii.large,
+}));
+
+const receiveButtonStyle = prepareNativeStyle(() => ({
+    width: 310,
```
Comment (Nodonisko) [inline suite-native/transactions/src/components/TransactionsEmptyState.tsx]: Are we sure we want fixed width here? Would be some kind of percentage better?

File: suite-native/navigation/src/components/TabBar.tsx (hunk 4 above)
Comment (juriczech) [reply suite-native/navigation/src/components/TabBar.tsx:42]: Same. I'm down to use this for smaller things that just need to get propagated across the app. 

For example I probably wouldn't use this for accounts and transactions handling, but stuff like this and discreet mode which we discussed recently, it's just super nice to use this for it. 

There is really no need to create action, reducer case, selector and default value in state, if you can do all that in one line with this. 

Hooked 😅

File: suite-native/transactions/src/components/TransactionsEmptyState.tsx (hunk 5 above)
Comment (juriczech) [reply suite-native/transactions/src/components/TransactionsEmptyState.tsx]: Changed to percentage. If there will be problem, we can always use fixed size.

File: suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx (hunk 2 above)
Comment (juriczech) [reply suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx]: It's still not 100% finished screen, but I feel like adding this functionality will help also QA with testing and it's been annoying me for some time that I have to delete all accounts if I want to delete just one. So here it is. Button color etc is being discussed in Figma with UX.

https://user-images.githubusercontent.com/36101761/200740339-085f4549-e14c-46bd-b467-0ad93034cc46.mov

File: suite-native/module-accounts/src/components/TransactionListHeader.tsx (hunk 3 above)
Comment (juriczech) [reply suite-native/module-accounts/src/components/TransactionListHeader.tsx]: yeah, good point. Used selector
